### PR TITLE
chore(all): Increase default test timeouts

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview
I'm tired of flaky tests. This is an attempt at preventing future flake.

### CODE REVIEW

**Review by eventualbuddha — APPROVED** [review]

File: frontends/bmd/cypress.config.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { defineConfig } from 'cypress'
 export default defineConfig({
   viewportHeight: 1080,
   viewportWidth: 1920,
+  defaultCommandTimeout: 8_000,
```
Comment (arsalansufi) [inline frontends/bmd/cypress.config.ts:6]: TIL about `_` as a number separator! I have a slight preference for reserving these separators for cases where there are greater than 3 zeros (e.g. 10_000) but using them more consistently is fine, too!

**Review by arsalansufi — APPROVED** [review]
Thanks for tackling this! Fingers crossed this does the trick

File: frontends/bmd/cypress.config.ts (hunk 1 above)
Comment (jonahkagan) [reply frontends/bmd/cypress.config.ts:6]: Ah yeah it does seem most useful with many zeros. I just got into the habit of using them all the time now. It grew on me :D

File: frontends/bmd/src/app_end_to_end.test.tsx [hunk 2]
```diff
@@ -468,9 +468,8 @@ it('MarkAndPrint end-to-end flow', async () => {
       name: 'Yes, Delete Election Data',
     })
   );
-  await waitFor(
-    () => expect(screen.queryByRole('alertdialog')).not.toBeInTheDocument(),
-    { timeout: 2000 }
```
Comment (arsalansufi) [inline frontends/bmd/src/app_end_to_end.test.tsx:473]: :100:  - thanks for removing these

**Pull request merged**